testing/functional/tests/cypress/integration/03-ministry/01-login-ministry-page.spec.ts

```diff
@@ -0,0 +1,23 @@
+import LoginMinistryObject from "../../page-objects/ministry-objects/LoginMinistryObject";
+import MinistryCustomCommand from "../../custom-command/ministry/MinistryCustomCommand";
+
+describe("Login Ministry Page", () => {
+  const loginMinistryObject = new LoginMinistryObject();
+  const ministryCustomCommand = new MinistryCustomCommand();
+
+  const url = Cypress.env("ministryURL");
+
+  beforeEach(() => {
+    cy.visit(url);
+  });
+
+  it("Verify that user able to login with a valid username and valid password.", () => {
+    ministryCustomCommand.loginMinistry();
+  });
+
+  it("Verify that clicking on the Log Off then it must be log out.", () => {
```

Comment:
How we are ensuring that the user is really logged out here?
Should it try to access the dashboard, for instance, and check if there is a redirect to the login page?

Reply:
After logged out it redirects to 'Welcome' page.
<img width="446" alt="image" src="https://user-images.githubusercontent.com/98920571/167674308-ecb3b89f-00da-49d0-aaaa-480fb0f0767b.png">
<img width="291" alt="image" src="https://user-images.githubusercontent.com/98920571/167674434-5b427126-dfeb-4154-92cb-6e839da9a6a7.png">
<img width="473" alt="image" src="https://user-images.githubusercontent.com/98920571/167674874-e160c4e1-1f9c-40a6-bd9f-be46e46694fd.png">